app/views/magic/check_your_email.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render PageTitle::View.new(i18n_key: "check_your_email") %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('authentication.check_your_email') %>
+    </h1>
+    <p class="govuk-body-l">
+      We’ve sent you an email. Click on the link to confirm your address and return to this service.
```

Comment:
Is this the content we're using elsewhere? I wonder if we should include something about "if you email is in the system we'll have sent you an email" sort of thing. There are a few reasons we wouldn't have sent an email e.g. if it doesn't exist or if the user hasn't logged in with signin before.